app.py

```diff
@@ -1,8 +1,14 @@
 import json
 
 from pandelytics import search
+from fastapi import FastAPI
 
-if __name__ == "__main__":
-    news = search.search("jhu", "Data")
-    if news:
-        print(json.dumps(news, indent=4))
+app = FastAPI()
+
+
+@app.get("/")
+async def root():
+    if __name__ == "__main__":
+        news = search.search("jhu", "Data")
```

Comment:
Can you verify that the news is a valid

Reply:
When I print it normally, it gives an output

Reply:
> When I print it normally, it gives an output

I also checked type(news) and it's dict

Reply:
Can you remove the if statement, just return the news object